code-of-conduct.md

```diff
@@ -0,0 +1,70 @@
+
+### Expected behavior
+
+All participants in our events and communications are expected to show respect and courtesy to others. All interactions should be professional regardless of platform: either online or in-person. In order to foster a positive and professional learning environment we encourage the following kinds of behaviors in all CCDL training events and platforms:
+
+- Use welcoming and inclusive language
+
+- Be respectful of different viewpoints and experiences
+
+- Gracefully accept constructive criticism
+
+- Focus on what is best for the community
+
+- Show courtesy and respect towards other community members
+
+Note: See the [four social rules](https://www.recurse.com/manual#sub-sec-social-rules) for further recommendations.
+
+### Unacceptable behavior
+
+Examples of unacceptable behavior by participants at any CCDL training event/platform include:
+
+- written or verbal comments which have the effect of excluding people on the basis of membership of any specific group
+
+- causing someone to fear for their safety, such as through stalking, following, or intimidation
+
+- violent threats or language directed against another person
+
+- the display of sexual or violent images
+
+- unwelcome sexual attention
+
+- nonconsensual or unwelcome physical contact
+
+- sustained disruption of talks, events or communications
+
+- insults or put downs
+
+- sexist, racist, homophobic, transphobic, ableist, or exclusionary jokes
+
+- excessive swearing
+
+- incitement to violence, suicide, or self-harm
+
+- continuing to initiate interaction (including photography or recording) with someone after being asked to stop
+
+- publication of private communication without consent
+
+If you at any time feel harassed or treated inappropriately please contact ccdl@alexslemonade.org.
+
+### Consequences of Unacceptable behavior
+
+Participants who are asked to stop any inappropriate behavior are expected to comply immediately. This applies to any CCDL training events and platforms, either online or in-person. If a participant engages in behavior that violates this code of conduct, the organizers may warn the offender, ask them to leave the event or platform.
+
+# License
+This document is based on the code of conduct written by [The Carpentries](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html#code-of-conduct-detailed-view), which is adapted from the guidelines by the [Django Project](https://www.djangoproject.com/conduct/enforcement-manual/), which was itself based on the [Ada Initiative](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Responding_to_reports) template. 
```

Comment:
You don't state what the license is here. Should it be CC-BY? I forget what license is used for the other guidelines you link to here. It would need to be compatible with those.

Reply:
Some of them are CC-BY and I think Ada Initiative on is CC0